src/browser/action.js

```diff
@@ -156,6 +167,16 @@ async function renderScripts() {
     }
 
     scriptsSection.appendChild(fieldset);
+
+    $(fieldset).find('.makeSpectrum').spectrum({
+      preferredFormat: "hex",
+      showInput: true,
+      showButtons: false,
+      allowEmpty: true
+    });
+    $(fieldset).find('.makeSpectrum').on('change.spectrum', (e) => {
+      writePreference(e);
+    })
```

Comment:
would be nice to save `.makeSpectrum` instead of calling the jQuery constructor twice

```suggestion
    const $makeSpectrum = $(fieldset).find('.makeSpectrum');
    
    $makeSpectrum.spectrum({
      preferredFormat: 'hex',
      showInput: true,
      showButtons: false,
      allowEmpty: true
    });
    $makeSpectrum.on('change.spectrum', (e) => {
      writePreference(e);
    });
```

also, can't you just pass in `writePreference` directly here?